css-env-1/Overview.bs

```diff
@@ -77,6 +77,53 @@ tho user agents may define additional undocumented [=environment variables=]:
 
 * Issue: define list of predefined vars
 
+Safe area inset variables {#safe-area-insets}
+------------------------------------------------------------------
+
+<pre class=propdef>
+	Name: safe-area-inset-top
+	Applies to: all elements
+	Value: <<length>>
+	Initial: 0
+	Inherited: yes
+	Media: visual
+	Animatable: no
+</pre>
+
+<pre class=propdef>
+	Name: safe-area-inset-right
+	Applies to: all elements
+	Value: <<length>>
+	Initial: 0
+	Inherited: yes
+	Media: visual
+	Animatable: no
+</pre>
+
+<pre class=propdef>
+	Name: safe-area-inset-bottom
+	Applies to: all elements
+	Value: <<length>>
+	Initial: 0
+	Inherited: yes
+	Media: visual
+	Animatable: no
+</pre>
+
+<pre class=propdef>
+	Name: safe-area-inset-left
+	Applies to: all elements
+	Value: <<length>>
+	Initial: 0
+	Inherited: yes
+	Media: visual
+	Animatable: no
+</pre>
+
+The safe area insets are four [=environment variables=] that define the top, right, bottom and
+left insets from the edge of the document. Page content that is in this safe area may be
```

Comment:
'document' seems wrong here - shouldn't it be something like 'viewport'?

Reply:
And I've been assuming the safe area is the bit inside the insets, so it I think it would be 'content that is outside this safe area' or 'content that is inside these insets'

Reply:
Sounds good - fixed.